src/components/Tabs/stories/Tabs.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Tabs from "..";
+import "../Tabs.css";
+
+export default {
+  title: "Components/Tabs",
+  component: Tabs,
+};
+
+export const Default = () => (
+  <div className="bg">
+    <h1>Tabs Demo</h1>
+    <Tabs>
+      <div label="Steps">
```

Comment:
Rather than this, it'd be amazing if your Tabs component also exported a component `Tab`, which simply render its children. That would let you set a `label` prop, through `prop-types` that would ensure that whoever uses the component would pass a label to the Tab. 

If this is not clear, please let me now and I'll show what I mean